r_examples/r_byo_r_algo_hpo/Dockerfile

```diff
@@ -1,17 +1,45 @@
-FROM ubuntu:16.04
+# Pull R Base image from Amazon Public ECR Gallery
+FROM public.ecr.aws/u6k6n4j8/r-base:latest
 
 MAINTAINER Amazon SageMaker Examples <amazon-sagemaker-examples@amazon.com>
 
 RUN apt-get -y update && apt-get install -y --no-install-recommends \
+    apt-utils \ 
     wget \
     r-base \
     r-base-dev \
+    libcurl4-openssl-dev \
     apt-transport-https \
+    libsodium-dev \
     ca-certificates
-
-RUN R -e "install.packages('plumber', repos='https://cloud.r-project.org')"
-RUN R -e "install.packages('https://cran.r-project.org/src/contrib/Archive/mda/mda_0.4-10.tar.gz', repos = NULL, type='source')"
-
+    
+    
+# install plumber and deps from https://packages.debian.org/sid/r-cran-plumber
+RUN apt-get install -y --no-install-recommends \
+    libjs-bootstrap \
+    libjs-jquery \
+    r-api-4.0 \
```

Comment:
Why doesn't the R base image have all of these R dependencies already? 
Or why don't we get a more loaded R image added to the public repo, so customers don't have to spend time installing common dependencies?

Reply:
The purpose of installing these dependencies manually was to show the user how they can create custom R images by installing custom libraries.